apps/qubit/modules/settings/actions/editAction.class.php

```diff
@@ -39,6 +39,10 @@ public function execute($request)
                     $this->getUser()->setFlash('notice', $this->updateMessage);
                 }
 
+                if (null !== $request->getFiles() && 'SettingsDiacriticsAction' === get_called_class()) {
```

Comment:
Why is the `get_called_class` check needed?

Reply:
I set this in case we add other settings in the future that requires calling getFiles as well. Should I leave that out for now since it's the only setting requiring getFiles?

Reply:
Oh, I see it now. This should be moved into the child class `processForm()` method then.